package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "react-native-camera",
   "description": "A Camera component for React Native. Also reads barcodes.",
-  "version": "3.17.0-patch-8",
+  "version": "3.19.0",
```

本体に追いついてから、またパッチしてるので、一応 suffix 付けたいです。

```suggestion
  "version": "3.19.0-9",
```